app/javascript/react/src/components/greetings.jsx

```diff
@@ -0,0 +1,20 @@
+import React, { useEffect } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { getGreetings } from './redux/slice'; // Update the correct path to your slice file
+
```

Comment:
- [ ] I can see that the getGreetings is inside the slice file which is inside the redux folder, a sister folder to the folder holding this greetings file, as a result, the path should go a step outside the folder and it requires double dots to take a step out of this current folder into the slice folder, I even encountered an error and I figured this to be the issue. just correct the path add `'../redux/slice'` and we are good to go.